src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenUsingStatementTests.cs

```diff
@@ -1491,7 +1433,11 @@ public void Dispose()
     }
 }
 ";
-            CreateCompilation(source).VerifyDiagnostics();
+            CreateCompilation(source).VerifyDiagnostics(
+                // (7,16): error CS1674: 'Program.MyManagedClass': type used in a using statement must be implicitly convertible to 'System.IDisposable' or have a public void-returning Dispose() instance method.
```

Comment:
>or have a public void-returning Dispose() instance method [](start = 153, length = 57)

The type does have a public instance `Dispose` method.

Reply:
I'm going to do an error message pass shortly, as everything is out of date.